package/src/components/MessageList/MessageList.tsx

```diff
@@ -1102,6 +1106,13 @@ const MessageListWithContext = <
     );
   }
 
+  const ItemSeparatorComponent = additionalFlatListProps?.ItemSeparatorComponent;
+  const WrappedItemSeparatorComponent = () => (
+    <View style={[shouldApplyAndroidWorkaround ? styles.invertAndroid : undefined]}>
+      {ItemSeparatorComponent ? <ItemSeparatorComponent /> : null}
+    </View>
+  );
```

Comment:
Can we put this in a useCallback, creating a component on every render would be bad

Reply:
Sure